core/src/main/java/io/grpc/internal/PickFirstLoadBalancerProvider.java

```diff
@@ -82,15 +83,10 @@ public ConfigOrError parseLoadBalancingPolicyConfig(Map<String, ?> rawLbPolicyCo
 
   private static Object getLbPolicyConfig(Map<String, ?> rawLbPolicyConfig) {
     Boolean shuffleAddressList = JsonUtil.getBoolean(rawLbPolicyConfig, SHUFFLE_ADDRESS_LIST_KEY);
-    if (enableNewPickFirst) {
+    if (isEnabledNewPickFirst()) {
       return new PickFirstLeafLoadBalancerConfig(shuffleAddressList);
```

Comment:
Grr... We need to ensure that the right config goes to the the right variation of the PF policy... but we can't actually do that without being consistent in our choice... which is then static.

Let's give up on this, at least for now.